game_event_player/tests.py

```diff
@@ -0,0 +1,90 @@
+from django.contrib.auth import get_user_model
+import pytest
+from django.db.utils import IntegrityError
+from django.core.exceptions import ValidationError
+from .models import GameEventPlayer
+from game_event.models import GameEvent
+from player.models import Player, BallGame
+from court.models import Court
+from decimal import Decimal
+from datetime import datetime
+
+
+@pytest.fixture
+def user():
+    return get_user_model().objects.create_user(
+        username='testuser',
+        password='testpass'
+    )
+
+
+@pytest.fixture
+def saved_player(user):
+    player = Player.objects.create(
+        user=user,
+        birth_date='1990-01-01',
+        favorite_ball_game=BallGame.Soccer
+    )
+    return player
+
+
+@pytest.fixture
+def saved_court():
+    court = Court.objects.create(x=Decimal('11'), y=Decimal('22'),
+                                 city='TEST_CITY', neighborhood='TEST_NEIGHBORHOOD', max_players=30)
+    return court
+
+
+@pytest.fixture
+def saved_game_event(saved_court):
+    game_event = GameEvent.objects.create(id=2, time=datetime.today(), level_of_game=3,
+                                          min_number_of_players=2, max_number_of_players=5, court_id=saved_court,
+                                          ball_game='Basketball')
+    return game_event
+
+
+@pytest.fixture
+def saved_game_event_player(saved_game_event, saved_player):
+    return GameEventPlayer.objects.create(game_event_id=saved_game_event, player_id=saved_player,
+                                          ball_responsible=False)
+
+
+@pytest.mark.django_db
+class TestGameEventPlayerModel:
+
+    def test_game_event_player_creation(self, saved_game_event, saved_player):
+        game_event_player = GameEventPlayer.objects.create(game_event_id=saved_game_event, player_id=saved_player,
+                                                           ball_responsible=False)
```

Comment:
Can you call `full_clean` here to show that it passes validation when the inputs are good.

Reply:
> Can you call `full_clean` here to show that it passes validation when the inputs are good?

Do I need to add the full_clean() after the assert commands?

Reply:
I'd suggest maybe to drop the assignment to `ball_responsible` so we can test the default value.

Reply:
> > Can you call `full_clean` here to show that it passes validation when the inputs are good?
> 
> Do I need to add the full_clean() after the assert commands?

```suggestion
        game_event_player = GameEventPlayer.objects.create(
                game_event_id=saved_game_event,
                 player_id=saved_player,
        ).full_clean()
```

Reply:
> > > Can you call `full_clean` here to show that it passes validation when the inputs are good?
> > 
> > 
> > Do I need to add the full_clean() after the assert commands?

When I call "full_clean()" function, if all checks pass, it returns 'None' (I read on chat gpt)
and we use the game_event_player after. Do I need to add "if is None" statement after do full_clean()?

Reply:
ok. So I suppose you'd need to add another test in which you will call full_clean and assert it's not throwing anything. I think we did similar stuff on the other PRs. See how it's done there.

Reply:
done

Reply:
I think we had a miscommunication here. What @Yuval-Vino suggested on the other thread is the way to go.